astrobee/survey_manager/survey_planner/src/survey_planner/command_astrobee.py

```diff
@@ -134,59 +136,87 @@ def __init__(self, robot_name):
         self.sock_input.bind(self.input_path)
         self.sock_input.listen(1)  # Listen for one connection
         self.sock_input_connected = False
+        self.sock_input_conn = None
```

Comment:
Might want to add some more comments explaining what's going on with these sockets

Reply:
added some comments :)